projects/cps-ui-kit/src/lib/components/cps-tree-autocomplete/cps-tree-autocomplete.component.ts

```diff
@@ -152,6 +171,13 @@ export class CpsTreeAutocompleteComponent
     }
   }
 
+  isActive() {
+    return (
+      this.isOpened ||
+      document.activeElement === this.treeAutocompleteInput?.nativeElement
```

Comment:
I'd suggest using document object from DI instead of global variable to support SSR

Reply:
Sure, thanks, I'll make this change as we start to consider potential SSG usecases.
Will use this DI token in other components as well.